public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_log_analytics/azure_log_analytics_datasource.ts

```diff
@@ -35,6 +35,7 @@ export default class AzureLogAnalyticsDatasource extends DataSourceWithBackend<
 > {
   url: string;
   baseUrl: string;
+  deepLinkUrl: string;
```

Comment:
nit: maybe call it `azurePortalUrl` to easier undestand what is it actually, since it's just a url to Azure Portal.